test/iron-overlay-behavior.html

```diff
@@ -133,14 +185,23 @@
           assert.isFalse(spy.called, 'overlay should not refit');
         });
 
-        test('open() triggers iron-resize', function(done) {
+        test('open() & close() trigger iron-resize', function(done) {
```

Comment:
I think this was refactored in https://github.com/PolymerElements/iron-overlay-behavior/pull/108 as well right?